recipe_sha: 4da918f1cf74dba3edb6847144651dd234c3260254f50d9aaca1227529f8061c  # The commit at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |-
  source activate  /opt/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
    Build:
  source activate  /opt/conda/conda-bld/_build_env


  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 141, in mamba_get_install_actions
      solution = solver.solve_for_action(_specs, prefix)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 244, in solve_for_action
      t = self.solve(specs)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 234, in solve
      raise RuntimeError("Solver could not find solution."  error_string)
  RuntimeError: Solver could not find solution.Mamba failed to solve:
   - bioconductor-dose >=3.26.0,<3.27.0
   - r-heatmaply
   - bioconductor-genomicranges >=1.52.0,<1.53.0
   - bioconductor-enrichplot >=1.20.0,<1.21.0
   - r-rcolorbrewer
   - bioconductor-sva >=3.48.0,<3.49.0
   - bioconductor-org.hs.eg.db >=3.17.0,<3.18.0
   - r-gplots
   - r-igraph
   - r-base 4.3.*
   - bioconductor-summarizedexperiment >=1.30.0,<1.31.0
   - r-ashr
   - bioconductor-deseq2 >=1.40.0,<1.41.0
   - bioconductor-s4vectors >=0.38.0,<0.39.0
   - r-shinydashboard
   - r-rcurl
   - r-dt
   - bioconductor-org.mm.eg.db >=3.17.0,<3.18.0
   - r-ggplot2
   - r-plotly
   - r-shiny
   - bioconductor-annotate >=1.78.0,<1.79.0
   - r-jsonlite
   - bioconductor-clusterprofiler >=4.8.0,<4.9.0
   - bioconductor-edger >=3.42.0,<3.43.0
   - r-shinyjs
   - r-shinybs
   - r-stringi
   - bioconductor-apeglm >=1.22.0,<1.23.0
   - bioconductor-annotationdbi >=1.62.0,<1.63.0
   - bioconductor-harman >=1.28.0,<1.29.0
   - r-colourpicker
   - bioconductor-limma >=3.56.0,<3.57.0
   - bioconductor-iranges >=2.34.0,<2.35.0
   - bioconductor-pathview >=1.40.0,<1.41.0
   - r-reshape2

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - nothing provides requested bioconductor-dose >=3.26.0,<3.27.0
  -   - nothing provides requested bioconductor-enrichplot >=1.20.0,<1.21.0
  -   - nothing provides requested bioconductor-sva >=3.48.0,<3.49.0
  -   - nothing provides requested bioconductor-org.hs.eg.db >=3.17.0,<3.18.0
  -   - nothing provides requested bioconductor-org.mm.eg.db >=3.17.0,<3.18.0
  -   - nothing provides requested bioconductor-annotate >=1.78.0,<1.79.0
  -   - nothing provides requested bioconductor-clusterprofiler >=4.8.0,<4.9.0
  -   - nothing provides requested bioconductor-annotationdbi >=1.62.0,<1.63.0
  -   - nothing provides requested bioconductor-pathview >=1.40.0,<1.41.0
  -

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
      sys.exit(main())
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
      call_conda_build(action, config)
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
      result = api.build(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
      return build_tree(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3078, in build_tree
      packages_from_this = build(metadata, stats,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2038, in build
      output_metas = expand_outputs([(m, need_source_download, need_reparse_in_env)])
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 787, in expand_outputs
      for (output_dict, m) in deepcopy(_m).get_output_metadata_set(permit_unsatisfiable_variants=False):
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 2524, in get_output_metadata_set
      conda_packages = finalize_outputs_pass(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/metadata.py", line 884, in finalize_outputs_pass
      fm = finalize_metadata(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 547, in finalize_metadata
      build_unsat, host_unsat = add_upstream_pins(m,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 409, in add_upstream_pins
      host_deps, host_unsat, extra_run_specs_from_host = _read_upstream_pin_files(m, 'host',
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 374, in _read_upstream_pin_files
      deps, actions, unsat = get_env_dependencies(m, env, m.config.variant,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 131, in get_env_dependencies
      actions = environ.get_install_actions(tmpdir, tuple(dependencies), env,
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 150, in mamba_get_install_actions
      raise err
  conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {MatchSpec("bioconductor-enrichplot[version='>=1.20.0,<1.21.0']"), MatchSpec("bioconductor-dose[version='>=3.26.0,<3.27.0']"), MatchSpec("bioconductor-annotationdbi[version='>=1.62.0,<1.63.0']"), MatchSpec("bioconductor-sva[version='>=3.48.0,<3.49.0']"), MatchSpec("bioconductor-clusterprofiler[version='>=4.8.0,<4.9.0']"), MatchSpec("bioconductor-annotate[version='>=1.78.0,<1.79.0']"), MatchSpec("bioconductor-org.hs.eg.db[version='>=3.17.0,<3.18.0']"), MatchSpec("bioconductor-org.mm.eg.db[version='>=3.17.0,<3.18.0']"), MatchSpec("bioconductor-pathview[version='>=1.40.0,<1.41.0']")}
# Last 100 lines of the build log.
